content/sensu-go/5.16/getting-started/sample-app.md

```diff
@@ -112,13 +112,15 @@ kubectl create -f go/deploy/sensu-backend.yaml
 
 Use Sensu role-based access control to create a `demo` namespace and a `demo` user.
 
-**1. Configure sensuctl to use the built-in `admin` user.**
+**1. Configure sensuctl to use the admin user.**
```

Comment:
I think the changes here work for now, but we'll likely need another update to this document once https://github.com/sensu/sensu-kube-demo/issues/25 is implemented.